cmake/TenzirRegisterPlugin.cmake

```diff
@@ -222,27 +222,12 @@ function (TenzirCompileFlatBuffers)
 
   foreach (schema IN LISTS FBS_SCHEMAS)
     get_filename_component(basename ${schema} NAME_WE)
-    # The hardcoded path that flatc generates.
-    set(output_file
-        "${output_prefix}/${FBS_INCLUDE_DIRECTORY}/${basename}_generated.h")
-    # The path that we want.
-    set(desired_file
+    # The expected output path.
+    set(output_path
         "${output_prefix}/${FBS_INCLUDE_DIRECTORY}/${basename}.hpp")
-    # Hackish way to patch generated FlatBuffers schemas to support our naming.
-    set("rename_${basename}"
-        "${CMAKE_CURRENT_BINARY_DIR}/flatbuffers_strip_suffix_${basename}.cmake"
```

Comment:
As we discussed last week in Slack, I think you need to drop the commit that changes this, as it doesn't work with older versions of FlatBuffers that we still rely on in our Dockerfile and CI.